### PR TITLE
Remove rets & set some spammy functions to trace level

### PR DESCRIPTION
**Pull request by Restioson**
WIP for now until I've addressed all of the three current targets of #2484

### CODE REVIEW

**Comment by klochowicz** [comment]
@Restioson could we merge this in stages? all 3 points in the issue linked are orthogonal, and it actually is easier to measure the effect if they're merged separately?

File: daemon/src/shared_protocol.rs [hunk 1]
```diff
@@ -50,13 +52,12 @@ pub(crate) async fn verify_cets(
...
         anyhow::Ok(())
     })
-    .in_current_span()
     .await??;
 
     Ok(())
 }
 
-#[instrument(skip_all)]
+#[instrument(target = "verify_crypto", level = "trace", skip_all)]
```
Comment (klochowicz) [inline daemon/src/shared_protocol.rs:60]: IMHO I would probably call them like module names, `shared_protocol` here, and `contract_setup` in the other case (no need to append the `_deprecated` there.

I'm not too fussed though, I'm happy as long as they're not printed out by default :)

**Review by klochowicz — APPROVED** [review]

File: daemon/src/shared_protocol.rs (hunk 1 above)
Comment (Restioson) [reply daemon/src/shared_protocol.rs:60]: > contract_setup in the other case (no need to append the _deprecated there.

How come? They do the same thing, don't they, so why different names?

Comment (klochowicz) [reply daemon/src/shared_protocol.rs:60]: fair enough. I was mostly pointing out that `shared_protocol` would not work for both, and this seemed to be the name liked by @luckysori most. As said above, I'm not too fussed; `verify_crypto` is a decent name too 👍

Comment (luckysori) [reply daemon/src/shared_protocol.rs:60]: I think `verify_crypto` is also fine!

**Review by luckysori — APPROVED** [review]

**Pull request merged**